### PR TITLE
added_difference_between_roles

### PR DESCRIPTION
**Pull request by YaTut1901**

### CODE REVIEW

**Review by andriylagun — APPROVED** [review]
Good job !

File: src/main/java/main/internet/shop/controller/AdminProductServiceController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import main.internet.shop.lib.Injector;
+import main.internet.shop.model.Product;
+import main.internet.shop.service.ProductService;
+
+public class AdminProductServiceController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/main/internet/shop/controller/AdminProductServiceController.java]:
```suggestion
public class GetAllProductsAdminController extends HttpServlet {
```

File: src/main/java/main/internet/shop/controller/ItemDeleteController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import main.internet.shop.lib.Injector;
+import main.internet.shop.model.Product;
+import main.internet.shop.model.ShoppingCart;
+import main.internet.shop.service.ProductService;
+import main.internet.shop.service.ShoppingCartService;
+
+public class ItemDeleteController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/main/internet/shop/controller/ItemDeleteController.java]:
```suggestion
public class DeleteProductFromCartController extends HttpServlet {
```

File: src/main/java/main/internet/shop/controller/OrdersAdminController.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import main.internet.shop.lib.Injector;
+import main.internet.shop.model.Order;
+import main.internet.shop.service.OrderService;
+
+public class OrdersAdminController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/main/internet/shop/controller/OrdersAdminController.java]:
```suggestion
public class GetAllOrdersAdminController extends HttpServlet {
```

File: src/main/java/main/internet/shop/controller/UserOrderController.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import main.internet.shop.lib.Injector;
+import main.internet.shop.service.OrderService;
+
+public class UserOrderController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/main/internet/shop/controller/UserOrderController.java]:
```suggestion
public class UserOrdersController extends HttpServlet {
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**